src/sentry/api/serializers/models/group.py

```diff
@@ -884,6 +884,7 @@ def serialize(
     ) -> BaseGroupSerializerResponse:
         result = super().serialize(obj, attrs, user)
         del result["annotations"]  # type:ignore
+        del result["assignedTo"]  # type:ignore
```

Comment:
With the current approach we may leak something else in the future if parent GroupSerializer changes.
Can we do the opposite: instead of deleting things that should not be there, we only keep those that are required by the frontend (https://github.com/getsentry/sentry/blob/master/static/app/views/sharedGroupDetails/index.tsx).
This way we will be more mindful when exposing data via shared issue view.